### PR TITLE
Implementation Layoutinformation loading, when exists

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- This PR should merge after GithubAction for SET implemented
- Load PlanPro file with layout.xml

### CODE REVIEW

File: java/bundles/org.eclipse.set.feature.validation/src/org/eclipse/set/feature/validation/modelloader/ModelLoaderImpl.java
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.feature.validation/src/org/eclipse/set/feature/validation/modelloader/ModelLoaderImpl.java:1]: dead code

File: java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/AbstractToolboxFile.java [hunk 1]
```diff
@@ -100,10 +94,6 @@ protected void loadResource(final Path path,
 			// Allow ppxml files with unknown features to be loaded
 			// by ignoring wrapped FeatureNotFoundExceptions
 			try {
-				// Disable find package throug namespace uri
-				resourceSet.getLoadOptions().put(
-						XMLResource.OPTION_USE_PACKAGE_NS_URI_AS_LOCATION,
-						Boolean.FALSE);
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/AbstractToolboxFile.java:106]: moved to toolboxmodel

File: java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/ZippedPlanProFileFormatService.java [hunk 2]
```diff
@@ -98,8 +99,9 @@ private EditingDomain createEditingDomain() {
 				final PlanProResourceFactoryImpl resourceFactory = new PlanProResourceFactoryImpl();
 				registry.getContentTypeToFactoryMap().put(extension,
 						resourceFactory);
+				final ToolboxModelService toolboxmodelService = new ToolboxModelServiceImpl();
 				resourceFactory.setToolboxModelServiceProvider(
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/ZippedPlanProFileFormatService.java:103]: ToolboxModelService shouldn't create new instance, when more resource load

File: java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/AbstractToolboxFile.java [hunk 3]
```diff
@@ -42,14 +43,7 @@ public abstract class AbstractToolboxFile implements ToolboxFile {
...
 
-	// IMPROVE: this Resource and Resource of EditingDomain have same content.
-	// Should we this Resouce here remove ?
-	private XMLResource resource;
-
-	@Override
-	public XMLResource getResource() {
-		return resource;
-	}
+	private HashMap<String, XMLResource> resources = new HashMap<>();
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/fileservice/AbstractToolboxFile.java:46]: now we need load more than one resource

**Review by pstuecker — APPROVED** [review]

**Pull request merged**